var/spack/repos/builtin/packages/py-networkit/package.py

```diff
@@ -20,12 +20,14 @@ class PyNetworkit(PythonPackage):
     homepage = "https://networkit.github.io/"
     url      = "https://pypi.io/packages/source/n/networkit/networkit-6.1.tar.gz"
 
+    version('8.0', sha256='36c30e894e835bf93f0aa0fb0b526758234e74318150820911e024ffe5ec1fd2')
     version('7.1', sha256='8609dc7a574a8a82d8880b8b1e3dfdd9c59ad67cd02135628e675c482fe98a96')
     version('7.0', sha256='eea4b5e565d6990b674e1c7f4d598be9377d57b61d0d82883ecc39edabaf3631')
     version('6.1', sha256='f7fcb50dec66a8253f85c10ff9314100de013c7578d531c81d3f71bc6cf8f093')
 
     # Required dependencies
```

Comment:
Can you also add:
```python
depends_on('python@3:', type=('build', 'run'))
```

Reply:
Thanks for the suggestion - I thought this would be covered by having a `PythonPackage` class.

Reply:
`PythonPackage` adds a Python dependency, but not a Python 3 dependency.